client/src/containers/backend/People/Makers/Edit.js

```diff
@@ -129,6 +132,20 @@ class MakersEditContainer extends PureComponent {
             name="attributes[avatar]"
             remove="attributes[removeAvatar]"
           />
+          <Form.HasMany
```

Comment:
Wondering why you use Form.HasMany on the maker side, and a dedicated container on the user side.